app/src/main/res/layout/fragment_event_list.xml

```diff
@@ -0,0 +1,38 @@
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This is exactly similar to the `event_list` xml except that it doesn't have data binding, which is going back in history as we will definitely use databinding in future to show and hide empty layout, so please don't edit it and it can be reused as is. This will also reduce the changes in repository